features/importing/archivist.feature

```diff
@@ -135,3 +185,12 @@ Feature: Archivist bulk imports
     When I check the 1st checkbox with id matching "importing_for_others"
     And I press "Import"
     Then I should see "We have notified the author(s) you imported works for. If any were missed, you can also add co-authors manually."
+
+  Scenario: Update the URL on an imported work with a redirect
+
+  # Given I have an Open Doors committee member "Ariana"
```

Comment:
Aha, that's because there were bits of OD tests in the other PR - I'll clean this out for now, and then can expand it once both are merged. Thanks for highlighting.